src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -17,7 +17,7 @@
     public static final String COMMAND_WORD = "find";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-sensitive) and displays them as a list with index numbers.\n"
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
```

Comment:
Good effort on updating the documentation, but try to update the header comments as well.